worlds/openrct2/__init__.py

```diff
@@ -0,0 +1,556 @@
+import math
+from typing import TextIO
+
+import worlds.LauncherComponents as LauncherComponents
+from BaseClasses import ItemClassification, Region
+from worlds.generic.Rules import add_rule
+
+from .Constants import *
+from .Items import *
+from .Locations import *
+from .Options import *
+from ..AutoWorld import World, WebWorld
+
+
+def launch_client() -> None:  # Shoutout to Serpent.ai for the launcher code!
+    from .Client import main
+    LauncherComponents.launch_subprocess(main, name="OpenRCT2Client")
+
+
+LauncherComponents.components.append(
+    LauncherComponents.Component(
+        "OpenRCT2 Client",
+        func=launch_client,
+        component_type=LauncherComponents.Type.CLIENT
+    )
+)
+
+
+class OpenRCT2WebWorld(WebWorld):
+    tutorials = []
+
+
+def get_previous_region_from_OpenRCT2_location(location_number: int):
+    if location_number == 0 or location_number == 1 or location_number == 2:
```

Comment:
```suggestion
    if location_number in [0, 1, 2]:
```
Or, if there are no location_numbers that are less than 0, you can do
`if location_number <= 2:`